### PR TITLE
fix: broken feature flags

### PR DESCRIPTION
**Pull request by Wodann**

### CODE REVIEW

File: crates/revm/src/inspector.rs [hunk 1]
```diff
@@ -6,7 +6,6 @@ use crate::{
 use auto_impl::auto_impl;
 use bytes::Bytes;
 
-#[cfg(feature = "std")]
```
Comment (rakita) [inline crates/revm/src/inspector.rs:9]: customprinter uses `println!` that is why it is behind `std`.

**Review by rakita — APPROVED** [review]
lgtm. just revert std for customprinter

**Comment by Wodann** [comment]
> lgtm. just revert std for customprinter

Gotcha. Then I'll add the `std` feature flag in the `pub use` underneath as well

**Comment by Wodann** [comment]
Re-inserted the `std` feature flag

**Pull request merged**